packages/spruce-next-helpers/src/skillskit/helpers/lang.ts

```diff
@@ -0,0 +1,115 @@
+import * as moment from 'moment'
+import Dinero from 'dinero.js'
+
+import { ICoreLocation } from '@sprucelabs/spruce-types'
+
+export default {
+	lang: {},
+	overrides: {},
+	async configure(langDir: string) {
+		this.lang = require(`${langDir}/default.js`)
+		try {
+			this.overrides = require(`${langDir}/overrides.js`)
+		} catch (err) {
+			console.info('No lang override specified.')
+		}
+	},
+	mixin(lang, overrides) {
+		this.lang = lang
+		this.overrides = overrides
+	},
+	getText(key: string, context: Record<string, any> = {}) {
+		const translations: Record<string, any> = {
```

Comment:
This will create an in-memory copy of the entire lang, plus overrides every time you call getText; better would be to look inside context, then overrides, then lang for key.